samples/spring-wiring-tests/src/main/java/wiring/eventsourcedentity/ValueMultiplied.java

```diff
@@ -0,0 +1,13 @@
+package wiring.eventsourcedentity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ValueMultiplied {
+    public final int value;
+
+    @JsonCreator
+    public ValueMultiplied(@JsonProperty Integer value) {
```

Comment:
```suggestion
    public ValueMultiplied(@JsonProperty int value) {
```

to align with `ValueIncreased`